Services/Common/OJS.Services.Common.Models/Contests/Results/ContestResultsModel.cs

```diff
@@ -0,0 +1,46 @@
+﻿namespace OJS.Services.Common.Models.Contests.Results;
+
+using AutoMapper;
+using OJS.Data.Models.Contests;
+using SoftUni.AutoMapper.Infrastructure.Models;
+using System.Linq;
+
+public class ContestResultsModel : IMapExplicitly
+{
+    public Contest Contest { get; set; } = new Contest();
```

Comment:
I see that you don't use much of the `Contest` entity. Better pass only the needed properties like ContestName, IsOfficial and ContestType as standalone properties here, instead of the whole data model.